docs/examples/index.md

```diff
@@ -1,8 +1,137 @@
 ---
-title: Basic usage
-sidebar_label: Basic usage
+sidebar_label: Examples
+title: 'Code examples'
 ---
 
-This section contains examples of how to use Apify Client for Javascript.
+## Passing an input to the Actor
 
-There is nothing here yet.
+The fastest way to get results from an Actor is to pass input directly to the `call` function.
+Input can be set up, can be passed to `call` function and the reference of running Actor (or wait for finish) is available in `runData` variable.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const actorClient = client.actor('apify/instagram-hashtag-scraper');
+
+const input = { hashtags: ['rainbow'], resultsLimit: 20 };
+
+// Run the Actor and wait for it to finish up to 60 seconds.
+// Input is not persisted for next runs.
+const runData = await actorClient.call(input, { waitSecs: 60 });
```

Comment:
How about adding a console log to the end of the example so if user runs it, there is some output printed - otherwise they won't know something actually happened

Reply:
Please check lines below where you can find additional content